vars/os_RedHat_7.yml

```diff
@@ -7,12 +7,12 @@ bootloader_update_command: grub2-mkconfig
 docker_version_map:
   "1.13":
     name: 'Docker'
-    package: docker-1.13.1
-    repo: rhui-REGION-rhel-server-extras/7Server/x86_64
+    package: docker-2:1.13.1-109*
+    repo: "Red Hat Enterprise Linux*7*Extra*RPMs)"
```

Comment:
Whats the reason for not spelling it out, is there some variability in the repo name?

Reply:
Yes, with a `yum repolist all` on RHEL 7.7:
```
repo id                                                               repo name                                                                                      status
rhui-rhel-7-server-rhui-extras-rpms/x86_64                            Red Hat Enterprise Linux 7 Server - Extras from RHUI (RPMs)                                    disabled
...
```

On RHEL 7.6:
```
repo id                                                                        repo name                                                                             status
rhui-REGION-rhel-server-extras/7Server/x86_64                                  Red Hat Enterprise Linux Server 7 Extra(RPMs)                                         disabled
...

```